Autopilot/Src/batterySensor.c

```diff
@@ -0,0 +1,9 @@
+void initADC()
+{
+
+}
+
+float getBatteryVoltage()
```

Comment:
This is not how a class gets implemented. https://stackoverflow.com/questions/10141079/how-do-i-split-classes-with-inheritance-up-into-different-h-cpp-files is the start of an explanation.

Also, since this is a c++ class, the file extension should be .cpp. This confused CMake in the past.


All this being said, I don't think you should bother with providing the blank implementation for the moment. Let the person actually implementing this module worry about that when the time comes.

Reply:
ok sounds good, I'll uncommit the file then